developer/tutorial/plugin-layouts-3.md

```diff
@@ -27,40 +29,77 @@ class CoreLayoutBeesknees extends Component {
     structure: PropTypes.object
   }
 
+  getComponent(name) {
+    try {
+      if (name) {
+        return assertComponent(name);
+      }
+    } catch (e) {
```

Comment:
You'll want to take a look at the style guide. We never use single-letter variable and we have a standard of `error` for this form.

Reply:
done